src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -152,14 +154,24 @@ static MultiTable * MultiSubqueryPushdownTable(Query *subquery);
  * plan and adds a root node to top of it. The  original query is only used for subquery
  * pushdown planning.
  *
+ * In order to support external parameters for the queries where planning
+ * is done on the original query, we need to replace the external parameters
+ * manually. To achive that for subquery pushdown planning, we pass boundParams
+ * to this function.
```

Comment:
Worth mentioning that we only need to do so, because we currently are unable to send parameters to workers?